align_system/utils/alignment_utils.py

```diff
@@ -27,7 +27,7 @@
 '''
 class AlignmentFunction(ABC):
     @abstractmethod
-    def __call__(self, kdma_values, target_kdmas, misaligned=False, kde_norm=None):
+    def __call__(self, kdma_values, target_kdmas, misaligned=False, kde_norm=None, probabilistic=False):
```

Comment:
Can we add `probabilistic` to the inputs description in the comment describing AlignmentFunction

Reply:
Fixed in [7f146b3](https://github.com/ITM-Kitware/align-system/pull/106/commits/7f146b39609076b556f247f42c774ce6863d2269)